crates/rpc/rpc/src/eth/api/block.rs

```diff
@@ -87,7 +87,7 @@ where
             let receipts = block
                 .body
                 .into_iter()
-                .zip(receipts.clone())
+                .zip(receipts.to_vec())
```

Comment:
I was sceptic using this, any other way to not use extra memory and a workaround only using arc.